aws_quota/check/route53.py

```diff
@@ -1,9 +1,25 @@
 from aws_quota.exceptions import InstanceWithIdentifierNotFound
 import typing
 import boto3
+import cachetools
 from .quota_check import InstanceQuotaCheck, QuotaCheck, QuotaScope
 
 
+@cachetools.cached(cache=cachetools.TTLCache(1, 600))
```

Comment:
https://github.com/gravitational/cloud/blob/0c16abc99802f8e95d3d86d31eaff42ec0c26952/deploy/fluxcd/src/platform/aws-quota-checker/deployment.yaml#L23-L24

We're running with ` --limits-check-interval=600` so our cache should be at least 1200 to cut calls in half?

I'd be game for 7200 so we're only looking querying limits every couple of hours.

Reply:
Same comment on line 13 and 18 as well. These are not high-frequency resource types.